RELEASE_PROCESS.md

```diff
@@ -27,43 +27,123 @@ In order to release, you will need the following accounts/permissions:
 
 ## THE PROCESS
 
-### Informing everyone of a pending release
-
-- [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
-
 ### Getting the build to be released into current with a minor/major version bump
 
-- Update the version file to the version to be released if minor version needs to be bumped rather than just the patch.
-- Update the release notes with the version, date and context.
-- Run all the tasks for update from FrequentTasks.doc
-- Make sure the omnibus build is into current channel.
-  (This is triggered by expeditor once the build and tests in buildkite go through ok once a commit is merged to master)
-- Make sure the habitat builds are passing.
-- For updating just the patch verion refer the the section below on #Preparing for the release
+- Create a new branch from the latest master branch. [QUESTION: or do we simply push a commit directly to master?]
```

Comment:
The major:minor bump can be done with Expeditor. On the PR for which to bump the major or minor version; add label Expeditor: Bump Version Minor or Expeditor: Bump Version Major. Double check that the correct label is applied since it will create artifacts etc with the applied label and is not the easiest to revert.

Reply:
Yes, but where does the PR come from?  And the PR's branch?  Do we checkout a new branch from master (that's what I assumed above)?